.github/workflows/docker-image-scan.yml

```diff
@@ -0,0 +1,48 @@
+name: Scan docker image
+#on:
+#  schedule:
+#    - cron: '30 12 * * *'
+
+on: [ push, workflow_dispatch ]
+  push:
+    branches: [ main, ap-2930-snyk-docker-scan ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main, ap-2930-snyk-docker-scan ]
+
+jobs:
+  scan-docker-image:
+    runs-on: ubuntu-latest
+    name: Scan docker image
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build docker image
+        run: |
+          docker build \
+            --tag hmrc-interface-application:scan \
```

Comment:
having done this on other projects I would suggest using the name of the repo `laa-hmrc-interface-service-api` here, just to aid clarify things for when you are trying to find the project on snyk's dashboard.

Having said that it is not appearing on snyk's dashboard, perhaps because the LAA's account it has reached its. limit 😬